.gitignore

```diff
@@ -14,3 +14,4 @@ tests/typings/dist/
 html-report
 coverage*.json
 lcov.info
+.vscode
```

Comment:
Generally, things like this we should add it to our global `.gitignore` (anything specific to the editors, or local filesystem, like `.DS_Store`).  I may have forgotten to do this in the past myself.

To set it up, check out [Create Global Git Ignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)